app/src/androidTest/java/org/xwiki/android/sync/auth/AuthenticatorActivityTest.java

```diff
@@ -29,8 +29,8 @@ public AuthenticatorActivity getActivity() {
         AuthenticatorActivity authenticatorActivity;
         Bundle bundle = new Bundle();
         //AccountAuthenticatorResponse response = new AccountAuthenticatorResponse(null);
-        String authTokenType = Constants.AUTHTOKEN_TYPE_FULL_ACCESS + "android.uid.system";
-        bundle.putString(AccountManager.KEY_ACCOUNT_TYPE, Constants.ACCOUNT_TYPE);
+        String authTokenType = Constants.Companion.getAUTHTOKEN_TYPE_FULL_ACCESS() + "android.uid.system";
```

Comment:
Why here you have replaced property by getter function?

Reply:
Replaced it by adding the const keyword. E.g: Constants.ACCOUNT_TYPE where ACCOUNT_TYPE in Constants/kt is: const val ACCOUNT_TYPE = "org.xwiki.android.sync"